amplipi/models.py

```diff
@@ -127,6 +127,19 @@ class BaseUpdate(BaseModel):
   name: Optional[str] = fields.Name
 
 
+class Rating(Enum):
```

Comment:
can you rename this to be a PandoraRating or something that makes it explicit what this type is?